DotaStats/Modules/MatchInfoModule/Presenter/MatchInfoConverter.swift

```diff
@@ -0,0 +1,241 @@
+import Foundation
+import UIKit
+
+protocol MatchInfoConverter: AnyObject {
+    func mainMatchInfo(rawMatchInfo: MatchDetail) -> MainMatchInfo
```

Comment:
С учетом того, что делает метод лучше бы его сигнатурку подправить до такой
```suggestion
    func mainMatchInfo(from rawMatchInfo: MatchDetail) -> MainMatchInfo
```
Для всех остальных подобных методов в протоколе можно сделать тоже самое